internal/util/update/resource-merge.go

```diff
@@ -224,7 +222,7 @@ func getSubDirsAndNonKrmFiles(root string) (sets.String, sets.String, error) {
 			}
 			return nil
 		}
-		match, err := r.ShouldSkipFile(info)
+		match, err := isMatch(matchFilesGlob, path)
```

Comment:
I would rename match variable to isKrmFile which is more readable.

Reply:
Updated the name of the function to `isKrmFile` and moved the glob slice to a package variable. I think this is cleaner.